modules/apps/object/object-admin-rest-impl/rest-openapi.yaml

```diff
@@ -1809,6 +1840,265 @@ paths:
+                        application/xml:
+                            schema:
+                                items:
+                                    $ref: "#/components/schemas/ObjectModel"
+                                type: array
+            tags: ["ObjectModel"]
+        post:
+            operationId: postObjectModel
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: "#/components/schemas/ObjectModel"
+                    application/xml:
+                        schema:
+                            $ref: "#/components/schemas/ObjectModel"
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+            tags: ["ObjectModel"]
+    "/object-models/by-external-reference-code/{externalReferenceCode}":
+        get:
+            operationId: getObjectModelByExternalReferenceCode
+            parameters:
+                - in: path
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+            tags: ["ObjectModel"]
+        put:
+            operationId: putObjectModelByExternalReferenceCode
+            parameters:
+                - in: path
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      type: string
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: "#/components/schemas/ObjectModel"
+                    application/xml:
+                        schema:
+                            $ref: "#/components/schemas/ObjectModel"
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+            tags: ["ObjectModel"]
+    "/object-models/by-external-reference-code/{externalReferenceCode}/object-definitions":
```

Comment:
We don't need this endpoint, since we can filter by objectModelExternalReferenceCode on the getObjectDefinitionsPage.

Reply:
Okay, agree :+1:

Reply:
Ok